app/main/views/suppliers.py

```diff
@@ -245,3 +246,63 @@ def update_contact_information(supplier_id, contact_id):
         abort(400, "Database Error: {0}".format(e))
 
     return jsonify(contactInformation=contact.serialize())
+
+
+@main.route('/suppliers/<supplier_id>/selection-answers/<framework_slug>',
+            methods=['GET'])
+def get_selection_questions(supplier_id, framework_slug):
+    application = SelectionAnswers.query.find_by_supplier_and_framework(
```

Comment:
This could only return 1 or 0 items, should we make it a model method instead of a query one, since there's no other meaningful filtering to be done on the results?